### ISSUE TITLE
HTAN2 Clinical Update Design and RFC

### ISSUE THREAD

**Issue by aclayton555** [ncihtan/htan2-data-model#11]
24-11/12 Sprint Kick-Off Discussion - What is needed for clinical data in HTAN2?

Kristen:
- Solid core clinical model that captures what CRDC needs (i.e. what standards they are landing on for the CRDC Submission Portal). 
- Need to get through clinical object classes, identify changes needed. Reflect on what was captured in HTAN1 and what we need to change.
- Need to decide on proposed changes and what format this should take
- Core model should have extensions for cancer-specific elements for new centers
- Include linkages to accession numbers with permissible values

Adam:

- Core model - what form does this take. A component like “HTAN Clinical Core” that is required, with optional additional templates. Currently, core is limited to demo and diagnosis, but is that sufficient?
- Want a core model to compliment how folks are generating data
- Definitely want core to be 100% required, and mappable to CRDC/CDS. 
- Coded attributes - has been a pain point in HTAN1. Kristen has thoughts on this...CDS is using uberon ontology - but no one uses this in practice. Propose we use ICDO code and populate primary diagnosis that goes with that code. But need to account if there are specific difference between code and diagnosis. Let CDS deal with mapping to uberon ontology

TL; DR -Kristen has been digging into this and will focus this sprint on design. Several conversation with CRDC stakeholders throughout December to align on standards in use and anticipated mapping.
